### DISCUSSION TITLE
River usage in production?

### DISCUSSION THREAD

**Discussion by raymondregrello** [riverqueue/river#555] [General]
Hi there! I'm curious if there are any stats or anecdotes you can share of River usage in production (e.g. how many companies, which companies, what kind of scale, which features are being used)?

**Comment by brandur** [comment]
We don't collect any analytics from River installations so it's not possible to say how broadly River is used, but the amount of repo activity does suggest that it has quite a few users.

We've heard from a couple companies with very well known names that they're using River, but even there, this was in private conversation, and not really something we can disclose without checking with them first.

**Reply by bgentry** [reply]
Adding to what @brandur said and without identifying any companies, there are users who’ve told us about production usage involving several million jobs per day. The range of companies we’ve heard from goes from small startups to large prominent ones, and even public companies.

I hope that helps!

**Reply by raymondregrello** [reply]
Totally understand there are no analytics collected, thank you for sharing the context from users! :)
